src/components/biggive-article-card/biggive-article-card.tsx

```diff
@@ -76,6 +80,15 @@ export class BiggiveArticleCard {
           <div class="content-wrap">
             {this.slug != '' ? <div class="slug text-colour-primary">{this.slug}</div> : null}
             {this.date != '' ? <div class="date">{this.date}</div> : null}
+
+            {this.mainImageUrl != '' ? (
+              <div class="main-image-container">
+                <div class="image-wrap">
+                  <img src={this.mainImageUrl} alt={this.mainImageAltText} title={this.mainImageAltText} />
```

Comment:
What's the reason for setting the title and alt text the same? They have different purposes as html attributes, are you sure we even want a title most of the time?

Reply:
Removed